src/Message/HostedCheckoutDecryptReturnUrlRequest.php

```diff
@@ -55,6 +55,7 @@ class HostedCheckoutDecryptReturnUrlRequest extends ExtendedAbstractRequest
 {
     /**
      * @return SimpleXMLElement
+     * @psalm-suppress PossiblyInvalidArrayAccess
```

Comment:
This was thrown an error due to `isset($urlParts['query']) ? (string) $urlParts['query'] : $ipn;` but we're checking that 'query' exists before using it so I decided to just suppress the issue. Same with the issue below.

Reply:
can you please add comments in the code saying why they're suppressed? helps ppl figure out if they can ever be removed in the future